### PR TITLE
Replace 'person_reid_msg_ptr_' with 'person_reid_topic_' [issue #125]

### LINKED ISSUE #125: 'person_reid_topic_' and 'segmented_object_topic_' were not declared.
### Version
dev-ov2020.3

### Reproduction link
https://github.com/intel/ros_openvino_toolkit/blob/dev-ov2020.3/dynamic_vino_lib/src/outputs/ros_service_output.cpp

### Steps to reproduce
run `catkin_make` in catkin_ws workspace.

### What is expected?
They have correct name.

### What is actually happening?
When I tried to build projects,  the console reports error:
```
error: ‘person_reid_topic_’ was not declared in this scope
   if (person_reid_topic_ != nullptr)

error: ‘segmented_object_topic_’ was not declared in this scope
   segmented_object_topic_ = nullptr;
```

### PR DESCRIPTION
**Pull request by Corsair-cxs**
fix #125
**What kind of change does this PR introduce?** 

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev-2020.3` branch for ros_openvino_toolkit, _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature 

**Other information:**

### CODE REVIEW

**Comment by LewisLiuPub** [comment]
- please don't add postfix "_ptr" to a pointer parameter.
- please follow the naming convension for class parameters. e.g. xxx_msg_ptr --> xxx_topic.

File: dynamic_vino_lib/src/outputs/ros_service_output.cpp [hunk 1]
```diff
@@ -86,7 +86,7 @@ void Outputs::RosServiceOutput::setServiceResponse(
     boost::shared_ptr<people_msgs::ReidentificationSrv::Response> response)
 {
   slog::info << "in Reidentification service::Response ...";
-  if (person_reid_topic_ != nullptr)
+  if (person_reid_msg_ptr_ != nullptr)
```
Comment (LewisLiuPub) [inline dynamic_vino_lib/src/outputs/ros_service_output.cpp:89]: parameter person_reid_msg_ptr violates the naming convension followed currently.
person_reid_topic is more aligned.

Comment (Corsair-cxs) [reply dynamic_vino_lib/src/outputs/ros_service_output.cpp]: Okey, I've replaced `person_reid_msg_ptr_` with `person_reid_topic_`. 
Thank you for your advise!

Comment (LewisLiuPub) [reply dynamic_vino_lib/src/outputs/ros_service_output.cpp]: thanks, good now.

**Pull request merged**